teleporter/teleporter.go

```diff
@@ -1,13 +1,219 @@
-// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// Copyright (C) 2022, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 package teleporter
 
-import "math/big"
+import (
+	"fmt"
+	"math/big"
+	"os"
 
-type Info struct {
-	Version                  string
-	FundedAddress            string
-	FundedBalance            *big.Int
-	MessengerDeployerAddress string
-	RelayerAddress           string
+	"github.com/ava-labs/avalanche-tooling-sdk-go/constants"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/evm"
+	"github.com/ava-labs/avalanche-tooling-sdk-go/utils"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+const (
```

Comment:
nit: should we use const
```
AvaLabsOrg         = "ava-labs"
TeleporterRepoName = "teleporter"
```
to generate releaseURL and so on ?

Reply:
addressing on add-relayer PR

Reply:
https://github.com/ava-labs/avalanche-tooling-sdk-go/pull/124/commits/c4c111b4aae46b48b69d542570918acc980d903a